src/block.js

```diff
@@ -3,7 +3,10 @@ export default function Block( { currentTotal, freeShippingFrom } ) {
 	const divWidth = ( progress > 100 ? 100 : progress ) + '%';
 	const divStyle = { width: divWidth };
 	const remaining = Number( freeShippingFrom - currentTotal ).toFixed( 2 );
-	const message = `Spend $${ remaining } more to get free US shipping.`;
+	const message =
+		remaining > 0
+			? `Spend only $${ remaining } more to get free US shipping.`
+			: `You've qualified for free shipping. Great job!`;
```

Comment:
Strings should be translatable.

Reply:
Thanks, Nadir. I just localized these strings.